src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -371,17 +371,17 @@
       <Link>Common\Unix\System.Native\Interop.GetTimestamp.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND '$(IsPartialFacadeAssembly)'!='true' AND '$(TargetGroup)' != 'uap'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' AND '$(TargetsWindows)' == 'true'">
     <Compile Include="$(CommonPath)\Interop\Windows\shell32\Interop.ShellExecuteExW.cs">
       <Link>Common\Interop\Windows\shell32\Interop.ShellExecuteExW.cs</Link>
     </Compile>
     <Compile Include="System\Diagnostics\Process.Win32.cs" />
     <Compile Include="System\Diagnostics\ProcessManager.Win32.cs" />
     <Compile Include="System\Diagnostics\ProcessStartInfo.Win32.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true' AND '$(IsPartialFacadeAssembly)'!='true' AND '$(TargetGroup)' == 'uap'">
-    <Compile Include="System\Diagnostics\Process.WinRT.cs" />
-    <Compile Include="System\Diagnostics\ProcessManager.WinRT.cs" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap'">
+    <Compile Include="System\Diagnostics\Process.Uap.cs" />
+    <Compile Include="System\Diagnostics\ProcessManager.Uap.cs" />
     <Compile Include="System\Diagnostics\ProcessStartInfo.WinRT.cs" />
```

Comment:
@jkotas I initially did but this one has a comment inside telling that some API is not available on WinRT so I reverted the change to this file. Given that should I still change it?